<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signalPicker</class>
 <widget class="QDialog" name="signalPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal Picker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Choose a Signal to Display</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="nodeCombo"/>
   </item>
   <item>
    <widget class="QComboBox" name="msgCombo"/>
   </item>
   <item>
    <widget class="QListWidget" name="signalList"/>
   </item>
   <item>
    <widget class="QPushButton" name="colorButton">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="selectButton">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
